test/helper.rb

```diff
@@ -183,6 +185,40 @@ def _new_client(options = {})
     end
   end
 
+  module Sentinel
+    include Generic
+
+    MASTER_PORT = PORT.to_s
+    SLAVE_PORT = '6382'
+    SENTINEL_PORT = '6400'
+    SENTINEL_PORTS = %w[6400 6401 6402].freeze
+    MASTER_NAME = 'master1'
+    LOCALHOST = '127.0.0.1'
+
+    def build_sentinel_client(options = {})
+      opts = { host: LOCALHOST, port: SENTINEL_PORT, timeout: TIMEOUT, logger: ::Logger.new(@log) }
+      Redis.new(opts.merge(options))
+    end
+
+    def build_slave_role_client(options = {})
+      _new_client(options.merge(role: :slave))
+    end
+
+    private
+
+    def _format_options(options = {})
```

Comment:
Why are these methods prefixed with an underscore? They are private, do they conflict with other methods in Generic?

Reply:
No, there are no conflicts. I think the underscore is meaningless too.

The `Generic` call the `#_new_client` method at [#setup](https://github.com/redis/redis-rb/blob/v4.1.0/test/helper.rb#L102). I think the private methods are overall need refactoring but this Pull Request is focused on the Sentinel tests.

* [Client](https://github.com/redis/redis-rb/blob/v4.1.0/test/helper.rb#L177-L183)
* [Distributed](https://github.com/redis/redis-rb/blob/v4.1.0/test/helper.rb#L197-L206)
* [Cluster](https://github.com/redis/redis-rb/blob/v4.1.0/test/helper.rb#L340-L350)